bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseActuatorHandler.java

```diff
@@ -187,9 +187,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             return;
         }
 
+        // check if the channel is linked otherwise do nothing
         String channelId = channelUID.getId();
         Channel channel = getThing().getChannel(channelId);
-        if (channel == null) {
+        if (channel == null || !isLinked(channelId)) {
```

Comment:
I recommend to use `channellUID` instead, its a little bit faster and needs less resources:

```suggestion
        if (channel == null || !isLinked(channelUID)) {
```